### ISSUE TITLE
validation dice score is always same during training and predict generate output with 0s. 

### ISSUE THREAD

**Issue by MNaseerSubhani** [milesial/Pytorch-UNet#442]
I have trained the model with custom dataset, the loss gradually decreasing but the validation score always the same. I tested the image with predict.py and got 0s output with blank mask.

**Comment by golangboy** [comment]
i stuck in the same problem as you.

**Comment by naguileraleal** [comment]
Hi. I have the same problem. Did you manage to solve it @MNaseerSubhani ?

**Comment by jiao1021** [comment]
我和你一样，我是自己用不同尺寸的数据集进行训练的，解决报错后能正常训练，但是DICE指标一直不变

**Comment by Ashlot2002** [comment]
请问你解决了吗，我也是相同的问题@jiao1021

**Comment by jiao1021** [comment]
> 请问你解决了吗，我也是相同的问题@jiao1021

我们修改了多次参数都无法解决。只能resize尺寸

**Comment by AKAxiaobai-123** [comment]
> > 请问你解决了吗，我也是相同的问题@jiao1021
> 
> 我们修改了多次参数都无法解决。只能resize尺寸
请问你们修改后的尺寸是多少呢

**Comment by YQF223006107** [comment]
I do some actions to solve this problem, for example data enhancement and picture enhancement. The problem comes form my data that are not clear so the net can not learn proficently the weight and bias.
